<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="IPText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PORTText">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>8888</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>PORT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
